### PR TITLE
[QA] Acceptance tests for 'obsolete images' behaviour and GlanceSync CLI

### PR DESCRIPTION
**Pull request by jframos**
#### Reviewers

@flopezag @chemaper 
#### Description

New acceptance test for:
- Obsolete images.
- GlanceSync CLI options.
#### Testing

Locally.

### CODE REVIEW

File: tests/acceptance/commons/glance_operations.py [hunk 1]
```diff
@@ -134,29 +134,41 @@ def create_image(self, image_glance_name, image_filename=None, container_format=
             image.update(properties=custom_properties)
         return image.id
 
-    def update_image_properties(self, image_id, custom_properties):
+    def update_image_properties(self, image_id, **kwargs):
         """
         Update the image properties of the given image_id
         :param image_id: Image ID
```
Comment (flopezag) [inline tests/acceptance/commons/glance_operations.py:140]: The description parameter does not match with the method parameters.

File: tests/acceptance/commons/glance_operations.py [hunk 2]
```diff
@@ -134,29 +134,41 @@ def create_image(self, image_glance_name, image_filename=None, container_format=
...
+            self.glance_client.images.update(image_id, name=kwargs['name'])
+
+        if "is_public" in kwargs:
+            __logger__.debug("Updating visibility of the image to '%s'", kwargs['is_public'])
+            self.glance_client.images.update(image_id, is_public=kwargs['is_public'])
+
+    def update_image_properties_by_name(self, image_name, **kwargs):
         """
         Update properties of all images found by the given name
         :param image_name: Name of the image to update (data)
```
Comment (flopezag) [inline tests/acceptance/commons/glance_operations.py:163]: Check params definition...

File: tests/acceptance/commons/glance_operations.py (hunk 1 above)
Comment (jframos) [inline tests/acceptance/commons/glance_operations.py:140]: The documented parameters should be among the given _**kwargs_ parameters. If not, they are not considered.

Comment (jframos) [inline tests/acceptance/commons/glance_operations.py:140]: documentation improved in a914afc

**Comment by jframos** [comment]
To be merge after #44

**Comment by flopezag** [comment]
LGTM

**Pull request merged**